### ISSUE TITLE
UnusedImport false positive when using import with an alias

### ISSUE THREAD

**Issue by piotrpolak** [pinterest/ktlint#356]
Hi,

When using `ktlint-maven-plugin:0.9.22:check` I get a `UnusedImport` false positive for the following code:

```
import com.github.tomakehurst.wiremock.client.WireMock.get as wiremockGet
import com.github.tomakehurst.wiremock.client.WireMock.verify as wiremockVerify

wiremockVerify(1, getRequestedFor(urlEqualTo("someUrl")))
```
I checked `ktlint-maven-plugin:0.9.22` dependencies and it uses `klint:0.31.0`, see https://github.com/gantsign/ktlint-maven-plugin/blob/345657aee10ad1c54fb051de2b087086759d35e4/pom.xml#L46

**Comment by NightlyNexus** [comment]
hm, it's working for me. you're using the `wiremockGet`, right?
can you make a minimum sample or test case for `NoUnusedImportsRuleTest`?

**Comment by piotrpolak** [comment]
I will prepare a sample. I noticed it works well when importing static values with aliases but fails when importing static methods.

**Comment by piotrpolak** [comment]
I extended one of the existing tests with the following methods:
```
   @Test
    fun testStaticMethodImportWithAlias() {
        assertThat(
            NoUnusedImportsRule().lint(
                """
                package ro.polak.controllertest

                import com.github.tomakehurst.wiremock.client.WireMock.get as wiremockGet
                import com.github.tomakehurst.wiremock.client.WireMock.post as wiremockPost
                import com.github.tomakehurst.wiremock.client.WireMock.verify as wiremockVerify

                class MyControllerTest : BaseControllerTest() {

                    fun testSomething() {
                        wiremockVerify(1, postRequestedFor(urlEqualTo("/oauth/token")))
                        wiremockVerify(1, getRequestedFor(urlEqualTo("url2")))
                        wiremockVerify(1, getRequestedFor(urlEqualTo("url3")))
                    }

                }
                """.trimIndent()
            )
        ).isEqualTo(
            listOf<LintError>()
        )
    }
```

And this is what I got

```
Running com.github.shyiko.ktlint.ruleset.standard.NoUnusedImportsRuleTest
Configuring TestNG with: org.apache.maven.surefire.testng.conf.TestNG652Configurator@5fcfe4b2
Tests run: 8, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 1.131 sec <<< FAILURE!
testStaticMethodImportWithAlias(com.github.shyiko.ktlint.ruleset.standard.NoUnusedImportsRuleTest)  Time elapsed: 0.013 sec  <<< FAILURE!
java.lang.AssertionError: 
Expecting:
 <[LintError(line=3, col=1, ruleId=no-unused-imports, detail=Unused import),
    LintError(line=4, col=1, ruleId=no-unused-imports, detail=Unused import)]>
to be equal to:
 <[]>
but was not.
	at com.github.shyiko.ktlint.ruleset.standard.NoUnusedImportsRuleTest.testStaticMethodImportWithAlias(NoUnusedImportsRuleTest.kt:298)


Results :

Failed tests:   testWiremock(com.github.shyiko.ktlint.ruleset.standard.NoUnusedImportsRuleTest): (..)
```

**Comment by NightlyNexus** [comment]
In that test case, where are `wiremockGet` and `wiremockPost` used in the source code?

**Comment by piotrpolak** [comment]
You are right, my bad. I thoughtlessly copied this piece of code from some larger block.

It turned out to be InteliJ Idea that does not detect those entries as unused (no highlighting, import optimization does not remove those imports).
